Dockerfile.redhat

```diff
@@ -409,6 +413,10 @@ LABEL supported-devices="CPU=1 GNA=1 GPU=${GPU} NVIDIA=${NVIDIA}"
 ARG RELEASE_BASE_IMAGE
 LABEL base-image=${RELEASE_BASE_IMAGE}
 
+WORKDIR /opt/intel/openvino/python
+COPY --from=build /opt/intel/openvino/python /opt/intel/openvino/python
```

Comment:
I wonder if we can make that copy depending on PYTHON_DISABLE setting

Reply:
we should add those python bindings to the binary package depending on the PYTHON_DISABLE
It should be added during creating binary package instead of here
all ovms related binaries should be in /ovms/lib